components/homme/src/test_src/dcmip16_wrapper.F90

```diff
@@ -544,8 +554,12 @@ subroutine dcmip2016_test1_forcing(elem,hybrid,hvcoord,nets,nete,nt,ntQ,dt,tl)
     elem(ie)%derived%FQ(:,:,:,2) = (rho_dry/rho)*dp*(qc-qc0)/dt
     elem(ie)%derived%FQ(:,:,:,3) = (rho_dry/rho)*dp*(qr-qr0)/dt
 
+
+    if (.not. case_planar_bubble ) then 
     qi=4; elem(ie)%derived%FQ(:,:,:,qi) = dp*ddt_cl
     qi=5; elem(ie)%derived%FQ(:,:,:,qi) = dp*ddt_cl2
+    endif
```

Comment:
can you add a variable, something like "use_toy_chemistry"?   that will make it clear what the new code changes are doing.   it could just be local, with dcmip2016_test1 calling this routine with .true., while the bubble calls this routine with .false.